### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
In this PR I created the markup for the Leaderboard app

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @Oscar-Espinoza, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn¿t have a README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by wale-prog — CHANGES_REQUESTED** [review]
## Hi @Oscar-Espinoza  :wave:  , 
# Status : Required Changes :recycle: 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Highlights
- No linters error :heavy_check_mark:
- Github flow :heavy_check_mark:
- repo professional :heavy_check_mark:
- Descriptive README included ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Please make sure that the entry and output files in your webpack.config.js file are configured properly to be sure that your application always has the latest build to serve.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "Simple leaderboard app that shows a list of the recent scores and updates through a refresh button. Built with Javascript, HTML, CSS and Webpack.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (wale-prog) [inline package.json:8]: Great work including the build command for your application, but it lacked a way to start your application. Your script may look like the following `"start": "webpack --config webpack.config.js --watch"`

**Review by wale-prog — CHANGES_REQUESTED** [review]
## Hi @Oscar-Espinoza  :wave:  , 
# Status: Required Changes :recycle: 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Highlights
- No linters error :heavy_check_mark:
- Github flow :heavy_check_mark:
- repo professional :heavy_check_mark:
- Descriptive README included ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Please make sure that the entry and output files in your webpack.config.js file are configured properly to be sure that your application always has the latest build to serve.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @Oscar-Espinoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**